js/interface.js

```diff
@@ -2537,7 +2537,8 @@ $('[data-save-rule]').click(function(event) {
     rule.require = requiredFields;
   }
 
-  var exclude = _.compact($('input[name="exclude"]').val().split(','));
+  // Spilt exclude columns list with ', ' as it contain space between columns names
```

Comment:
1. Typo in "Split"
2. I suggest using functional programming to split by `,` and then `trim` the results

```js
array.split(',').map(s => s.trim())
```

cc @tonytlwu

Reply:
@squallstar changes added in next commit thank you for inputs

Reply:
@SMohol there's still a typo in your comment, but let's please just remove the comment entirely as it doesn't make sense anymore.

Reply:
while testing this column list i got ' ,  ' instead of ',' that is why added a comment accordingly to rectify but removing it now.